collections/src/main/scala/strawman/collection/IterableOnce.scala

```diff
@@ -80,6 +99,33 @@ final class IterableOnceExtensionMethods[A](private val it: IterableOnce[A]) ext
 
   @deprecated("Use .iterator().find instead of .find on IterableOnce", "2.13.0")
   def find(p: A => Boolean): Option[A] = it.iterator().find(p)
+
+  @deprecated("Use .iterator().foldLeft instead of .foldLeft on IterableOnce", "2.13.0")
+  @`inline` def foldLeft[B](z: B)(op: (B, A) => B): B = it.iterator().foldLeft(z)(op)
+
+  @deprecated("Use .iterator().foldRight instead of .foldLeft on IterableOnce", "2.13.0")
+  @`inline` def foldRight[B](z: B)(op: (A, B) => B): B = it.iterator().foldRight(z)(op)
+
+  @deprecated("Use .iterator().foldLeft instead of /: on IterableOnce", "2.13.0")
+  @`inline` final def /: [B](z: B)(op: (B, A) => B): B = foldLeft[B](z)(op)
+
+  @deprecated("Use .iterator().foldRight instead of :\\ on IterableOnce", "2.13.0")
+  @`inline` final def :\ [B](z: B)(op: (A, B) => B): B = foldRight[B](z)(op)
+
+  @deprecated("Use .iterator().map instead of .map on IterableOnce or consider requiring an Iterable", "2.13.0")
+  def map[B](f: A => B): IterableOnce[B] = it match {
+    case it: Iterable[_] => it.asInstanceOf[Iterable[A]].map(f)
+    case _ => it.iterator().map(f)
+  }
+
+  @deprecated("Use .iterator().flatMap instead of .flatMap on IterableOnce or consider requiring an Iterable", "2.13.0")
+  def flatMap[B](f: A => IterableOnce[B]): IterableOnce[B] = it match {
+    case it: Iterable[_] => it.asInstanceOf[Iterable[A]].flatMap(f)
+    case _ => it.iterator().flatMap(f)
+  }
+
+  @deprecated("Use .to on Iterator or Iterable", "2.13.0")
+  @`inline` def to[C](factory: Factory[A, C]): C = factory.fromSpecific(it)
```

Comment:
Right. I probably added it because all of these methods on `IterableOnce` are so convenient. But with `Factory.from` generally taking an `IterableOnce` we now have a good replacement.